third-party/fbson/FbsonDocument.h

```diff
@@ -125,9 +125,12 @@ class FbsonDocument {
     uint8_t ver_;
   } header_;
 
-  char payload_[0];
+  char payload_[1];
```

Comment:
At some point it would be good to contribute this back to MySQL project, since this file was copied from there. Please add a TODO into the COMMIT.md file in this directory.